protractor/e2e/integration/home-work2.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {SecondPage} from "../support/secondPage";
+import {BlogPage} from "../support/blog-page.po";
+import {element} from "protractor";
+
+describe('home work 2', () => {
+
+    const secondPage = new SecondPage();
+    const blogPage = new BlogPage();
+
+    fit('check open article', async () => {
```

Comment:
don't forget to remove "fit" from your tests before pushing it to the repository. Just imagine, that you have 1000 tests in your app and now in your CI will be only 1 test ran:-)